main/states/led.py

```diff
@@ -12,6 +14,10 @@ class LED_COLOR:
 
     def __init__(self, num_led, global_brightness=MAX_BRIGHTNESS,
                  order='rgb', bus=0, device=1, max_speed_hz=8000000):
+        output = subprocess.check_output(["aplay", "-l"]).decode(sys.stdout.encoding)
+        self.driver_installed  = output.find("seeed") != -1
+        if (not self.driver_installed ):
```

Comment:
Don't leave space between `driver_install` and `)`.

Python coding style: https://www.python.org/dev/peps/pep-0008/?#whitespace-in-expressions-and-statements

Reply:
okie will make changes